docs/cli/cli-spans.md

```diff
@@ -0,0 +1,71 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CLI
+--->
+
+# Semantic Conventions for CLI (Command Line Interface) programs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply when instrumenting CLI programs.
+A CLI span is defined in terms of [process attributes][ProcessRegistry].
+
+The **span name** SHOULD be of the format `{process.executable.name}`. In case this is
+not informative enough because of interpreters calling other files, e.g. `python script.py` or in
+cases of subcommands, e.g. `flask run mycommand` instrumentation MAY provide a more
+descriptive span name providing a stringified version of `{process.command_args}` if appropriate.
+
+> **Warning**
+> The `process.command_args` value can have high cardinality in case of parameters passed by the user.
+> It is recommended to strip parameters if there is enough domain knowledge for being able to do so.
```

Comment:
I'd rephrase it to

```suggestion
The span name SHOULD, when possible, match low-cardinality representation of command line. All variable parameters SHOULD be replaced with a `?` placeholder.

If instrumentation does not parse and sanitize command line, it SHOULD use set span status to `{process.executable.name}`. 

Instrumentations that have additional context about executed commands MAY use different span name format and SHOULD document it.
```

This would be a bit more prescriptive, would almost match what we do in databases + we span name should have low-ish cardinality according to the spec.

Reply:
I don't feel ? would as natural as in db here.  Something like this below?

```
The span name SHOULD be set to `{process.executable.name}`. 
Instrumentations that have additional context about executed commands MAY use a different low-cardinality span name format and SHOULD document it.
```

Reply:
Looks good!